config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits.yaml

```diff
@@ -81,6 +81,28 @@ presubmits:
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api
       testgrid-tab-name: pr-verify
+  - name: pull-cluster-api-verify-external-links
+    optional: true
+    decorate: true
+    path_alias: sigs.k8s.io/cluster-api
+    always_run: true
+    labels:
+      preset-service-account: "true"
+    skip_branches:
+    - gh-pages
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20200802-ff98b58-1.18
+        command:
+        - "runner.sh"
+        - "make"
+        - "verify-book-links"
+        resources:
+          requests:
+            cpu: 7300m
```

Comment:
Do we need this much CPU for this job?

Reply:
I have no idea! Probably not, I just copied it from the other adjacent jobs. I see `apidiff` has no resource requests, so I'll drop it here and see what happens.